src/app/features/languageSlice.ts

```diff
@@ -0,0 +1,28 @@
+import { PayloadAction, createSlice } from '@reduxjs/toolkit';
+import i18n from '../../i18n/i18n';
+import { eLanguage } from '../../interfaces/index';
+
+interface ILanguageState {
+    currentLanguage: eLanguage;
```

Comment:
may besome other meaningful name than elanguage?

Reply:
Resolved in 68d3167516aa920351ce775de2d84003ce48c784